yarn.lock

```diff
@@ -1134,6 +1134,11 @@
     expect "^28.0.0"
     pretty-format "^28.0.0"
 
+"@types/js-beautify@1.13.3":
```

Comment:
Can you rebase and then run `yarn`? This shouldn't be included.

Reply:
yes! sorry about that

Reply:
shouldn't https://github.com/zenorocha/react-email/pull/107/files be merged first?

Reply:
No worries 😊